example/rnn/cudnn_lstm_bucketing.py

```diff
@@ -76,12 +78,15 @@ def train(args):
     if args.stack_rnn:
         stack = mx.rnn.SequentialRNNCell()
         for layer in range(args.num_layers):
+            dropout = 0.0
+            if layer < (args.num_layers - 1):
+                dropout = args.dropout
             stack.add(mx.rnn.FusedRNNCell(args.num_hidden, num_layers=1,
-                    mode='lstm', prefix='lstm_%d'%layer,
+                    mode='lstm', prefix='lstm_%d'%layer, dropout=dropout,
```

Comment:
@szha Let's do the following:
change dropoutcell to normal cell instead of modifier. it's ok to break interface since it wasn't working anyway.

then modify unfuse to insert dropout in between